scripts/testdata/satellite-config.yaml.lock

```diff
@@ -400,6 +400,9 @@ identity.key-path: /root/.local/share/storj/identity/satellite/identity.key
 # how many orders to batch per transaction
 # orders.settlement-batch-size: 250
 
+# how stale the node selection cache can be
+# overlay-cache-config.staleness: 3m0s
```

Comment:
Command line flags having "config" in the name look really weird.

Reply:
lol good point, will fix

Reply:
Maybe `overlay.node-selection-cache.staleness`, or `overlay.cache.staleness`?